header.go

```diff
@@ -992,6 +1111,10 @@ func (h *ResponseHeader) setSpecialHeader(key, value []byte) bool {
 			// Transfer-Encoding is managed automatically.
 			return true
 		}
+		if caseInsensitiveCompare(strTrailer, key) {
```

Comment:
```suggestion
		} else if caseInsensitiveCompare(strTrailer, key) {
```

Reply:
The previous judgement has a `return`, would it be clearer not to write `else` ?

Reply:
Behavior wise it doesn't matter, but it's better to follow the same style as the code around it which does use `else`.